docs/tutorials/github_actions.md

```diff
@@ -56,10 +56,13 @@ Push to master and that's it.
 
 Once the new tag is created, triggering an automatic publish command would be desired.
 
-In order to do so, the first two secrets need to be added with the information
-of our pypi account.
+In order to do so, the crendetial needs to be added with the information of our PyPI account.
 
-Go to `Settings > Secrets > Add new secret` and add the secrets: `PYPI_USERNAME` and `PYPI_PASSWORD`.
+Instead of using username and password, we suggest using [api token](https://pypi.org/help/#apitoken) generated from PyPI.
+
+After generate api token, use the token as the PyPI password and `__token__` as the username.
```

Comment:
If I read the [docs](https://packaging.python.org/guides/distributing-packages-using-setuptools/#create-an-account) and the [twine github action](https://github.com/pypa/gh-action-pypi-publish/blob/master/twine-upload.sh#L7) correctly, the password should have a `pypi-` prefix.

Reply:
Yes, but it's included as you generate the api token.

Reply:
Cool